src/components/LoadingIndicator/LoadingIndicator.stories.tsx

```diff
@@ -177,12 +178,15 @@ export const NoOverlay: Story<ILoadingIndicatorProps> = (args) => {
 		</LoadingIndicator>
 	);
 };
-NoOverlay.storyName = 'No Overlay';
+NoOverlay.args = {
+	hasOverlay: false,
+	isLoading: true,
+};
 
-/* Light Overlay */
-export const LightOverlay: Story<ILoadingIndicatorProps> = (args) => {
+/* Dark Overlay */
```

Comment:
I don't think we need to have comments like this since it's explicitly called DarkOverlay.  Just a preference but I will go with however the team feels.

Reply:
Agreed. I think we can do away with many of these old stories entirely in another round of cleanup now that SB6 has the interactive controls. I changed it to dark since light was the default. Maybe the title is slightly helpful when the story is a camelCase mouthful like : 
```/* Selected Index As Prop */
export const SelectedIndexAsProp: Story<IRadioGroupProps> = (args) => { ...
```